pages/ksphere/kommander/1.1/install-airgapped/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: layout.pug
+navigationTitle: Install air-gapped
+title: Install Kommander air-gapped
+menuWeight: 35
+excerpt: Install Kommander in an air-gapped environment
+enterprise: false
+---
+
+This document is meant to outline steps required to install kommander in an airgapped environment. It uses documentation of airgapped Konvoy installation as basis and outlines only the steps required to get Kommander running on top of an airgapped Konvoy cluster.
+
+# Naming
+
+Across this document, some terms are used which are defined here:
+* management cluster - konvoy cluster that is going to be running kommander
+* attached cluster - cluster, be it konvoy or non-konvoy one, which are going to be attached to the management cluster
+* docker registry - a registry from which all clusters download docker images used by installation
+
+# Before you begin
+
+Before installing, verify that your environment meets the following basic requirements:
+
+* a docker registry should already contain all the docker images required for installation and this includes also kommander images. The `konvoy_air_gapped.tar.bz2` tarball will contain the required artifacts.
+
+* connectivity with clusters that are going to be attached to the management
```

Comment:
yes, the internal LBs created in the attached clusters, especially those of the federated resources, might need to be accessible from the management cluster.